huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go

```diff
@@ -0,0 +1,582 @@
+				Computed:    true,
+				Description: `Indicates the name of updater.`,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the creation time of the data standard.`,
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the latest update time of the data standard.`,
+			},
+			"new_biz": {
+				Type:        schema.TypeList,
+				Elem:        dataStandardNewBizSchema(),
+				Computed:    true,
+				Description: `Indicates the biz info of manager.`,
+			},
+		},
+	}
+}
+
+func dataStandardValueSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"fd_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the name of the data standard attribute.`,
+			},
+			"fd_value": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `Specifies the value of the data standard attribute.`,
+			},
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of the stand element.`,
+			},
+			"fd_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of the data standard attribute.`,
+			},
+			"directory_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the directory ID that the attribute belongs to.`,
+			},
+			"row_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of data standard.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the status of the data standard.`,
+			},
+			"created_by": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the name of creator.`,
+			},
+			"updated_by": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the name of updater.`,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the creation time of the data standard.`,
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the latest update time of the data standard.`,
+			},
+		},
+	}
+	return &sc
+}
+
+func dataStandardNewBizSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of the new biz.`,
+			},
+			"biz_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the type of the new biz.`,
+			},
+			"biz_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of data standard.`,
+			},
```

Comment:
According the description of the parameter `biz_type`, does the descriptions of the parameter `id` and `biz_id` are comfused?